src/docfx/Errors.cs

```diff
@@ -82,6 +82,9 @@ public static Error LinkIsDependency(Document relativeTo, Document file, string
         public static Error AbsoluteFilePath(Document relativeTo, string path)
             => new Error(ErrorLevel.Warning, "absolute-file-path", $"File path cannot be absolute: '{path}'", relativeTo.ToString());
 
+        public static Error H1NotFound(Document file)
+            => new Error(ErrorLevel.Warning, "h1-not-found", $"Cannot find H1 with `#` or there are some non-comments block between yaml-header and H1", file.ToString());
```

Comment:
consider using `heading-not-found` since we don't want to tie the error code with just h1